.circleci/config.yml

```
@@ -355,6 +353,20 @@ jobs:
             docker push ${DOCKERHUB_REPO}:build_test
             docker push ${DOCKERHUB_REPO}:build_ui
             docker push ${DOCKERHUB_REPO}:latest
+  cirrus_deploy:
+    working_directory: ~/cirrus
+    machine:
+      image: ubuntu-2004:2023.04.2 # Ubuntu 20.04, Docker v20.10.24, Docker Compose v2.17.2
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - deploy:
+          name: Deploy to latest
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            make cirrus_build cirrus_test
```

Should this be cirrus_build_prod maybe? It looks like that's what actually builds and tags as `cirrus:deploy` based on the Makefile. Also if we aren't pushing the test image and this job requires `Check Cirrus` I don't think we need to include the test here.
```suggestion
            make cirrus_build_prod
```